linux/task.go

```diff
@@ -18,15 +19,21 @@ type Task struct {
 	pid       int
 	shim      *client.Client
 	namespace string
+	cg        cgroups.Cgroup
 }
 
-func newTask(id, namespace string, pid int, shim *client.Client) *Task {
+func newTask(id, namespace string, pid int, shim *client.Client) (*Task, error) {
+	cg, err := cgroups.Load(cgroups.V1, cgroups.PidPath(pid))
```

Comment:
This should be added to `runtime.loadTask()` too

Reply:
done